sdk/ml/azure-ai-ml/azure/ai/ml/_arm_deployments/arm_templates/workspace_base.json

```diff
@@ -451,29 +451,29 @@
                 "description": "Feature store online store connection target"
             }
         },
-        "materialization_identity_client_id": {
-            "type": "string",
-            "defaultValue": "",
-            "metadata": {
-                "description": "Feature store materialization credential client id"
-            }
-        },
         "materialization_identity_resource_id": {
             "type": "string",
             "defaultValue": "",
             "metadata": {
-                "description": "Feature store materialization credential resource id"
+                "description": "Feature store materialization identity resource id"
             }
         },
-        "setup_materialization_store": {
+        "grant_materialization_identity_permissions": {
```

Comment:
If remove "setup_materialization_store", would it break previous version?

Reply:
I don't think it will break, not it always set up offline store except if offline store target is already provided.